cpp/src/arrow/sparse_tensor.h

```diff
@@ -29,42 +29,58 @@ namespace arrow {
 // ----------------------------------------------------------------------
 // SparseIndex class
 
-/// \brief EXPERIMENTAL: Sparse tensor format enumeration
 struct SparseTensorFormat {
+  /// EXPERIMENTAL: The index format type of SparseTensor
   enum type { COO, CSR };
 };
 
-/// \brief EXPERIMENTAL: The base class for representing index of non-zero
-/// values in sparse tensor
+/// \brief EXPERIMENTAL: The base class for the index of a sparse tensor
+///
+/// SparseIndex are used to represent where is non-zero values in a
```

Comment:
This is less clear than it could be, maybe rewrite as "SparseIndex describes where the non-zero elements are within a SparseTensor."

Reply:
I rewrote the sentence.